babel.config.js

```diff
@@ -1,20 +1,20 @@
-{
-  "env": {
-    "test": {
-      "presets": [
-        "@babel/preset-env"
-      ]
-    },
-    "build": {
-      "presets": [
-        [
-          "@babel/preset-env",
-          {
-            "modules": false,
-            "loose": true
-          }
-        ]
-      ]
-    }
-  }
-}
+module.exports = {

```

Comment:
Otherwise it causes tests compilation on Windows

Reply:
why we have to change file name && code here?

Reply:
It's strange, but babel v7+ doesn't work with .babelrc on Windows (I tested it on Windows 10) - only with babel.config.js. JS files just stops transpiling and all tests fails because of syntax errors on import statements. You can test this configuration - it is completely same.